packages/mermaid/src/docs/community/development.md

```diff
@@ -1,358 +1,162 @@
-- followed by a _slash_ (which helps to group like types together in many git tools)
-- followed by the **issue number**
-- followed by an _underscore_ ('\_')
-- followed by a short text description (but use dashes ('-') or underscores ('\_') instead of spaces)
-
-If your work is specific to a single diagram type, it is a good idea to put the diagram type at the start of the description. This will help us keep release notes organized: it will help us keep changes for a diagram type together.
-
-**Ex: A new feature described in issue 2945 that adds a new arrow type called 'florbs' to state diagrams**
-
-`feature/2945_state-diagram-new-arrow-florbs`
-
-**Ex: A bug described in issue 1123 that causes random ugly red text in multiple diagram types**
-`bug/1123_fix_random_ugly_red_text`
-
-### 2. Write Tests
+<div v-if="selectedPlatform === 'native'">
+</div>
 
-Tests ensure that each function, module, or part of code does what it says it will do. This is critically
-important when other changes are made to ensure that existing code is not broken (no regression).
+<div v-if="selectedPlatform === 'docker'">
 
-Just as important, the tests act as _specifications:_ they specify what the code does (or should do).
-Whenever someone is new to a section of code, they should be able to read the tests to get a thorough understanding of what it does and why.
+### Make `./run` executable
 
-If you are fixing a bug, you should add tests to ensure that your code has actually fixed the bug, to specify/describe what the code is doing, and to ensure the bug doesn't happen again.
-(If there had been a test for the situation, the bug never would have happened in the first place.)
-You may need to change existing tests if they were inaccurate.
+For development using Docker there is a self-documented `run` bash script, which provides convenient aliases for `docker compose` commands.
 
-If you are adding a feature, you will definitely need to add tests. Depending on the size of your feature, you may need to add integration tests.
+Ensure `./run` script is executable:
 
-#### Unit Tests
-
-Unit tests are tests that test a single function or module. They are the easiest to write and the fastest to run.
-
-Unit tests are mandatory all code except the renderers. (The renderers are tested with integration tests.)
-
-We use [Vitest](https://vitest.dev) to run unit tests.
-
-You can use the following command to run the unit tests:
-
-```sh
-pnpm test
-```
-
-When writing new tests, it's easier to have the tests automatically run as you make changes. You can do this by running the following command:
-
-```sh
-pnpm test:watch
+```bash
+chmod +x run
 ```
 
-#### Integration/End-to-End (e2e) tests
-
-These test the rendering and visual appearance of the diagrams.
-This ensures that the rendering of that feature in the e2e will be reviewed in the release process going forward. Less chance that it breaks!
-
-To start working with the e2e tests:
+::: tip
```

Comment:
Use ```tip, not the `:::` one.

Reply:
I faced some issues with it. Only `:::` was working properly. I will try it again. Perhaps the problem was with its content. I needed a code block inside

Reply:
Done